app/models/manageiq/providers/nuage/manager_mixin.rb

```diff
@@ -81,10 +81,13 @@ def verify_credentials(auth_type = nil, options = {})
 
   def event_monitor_options
     @event_monitor_options ||= begin
+      amqp_auth = authentications.detect { |a| a.authtype == 'amqp' }
```

Comment:
Is there a way to get into this without `amqp` auth type?

Reply:
We assume that if amqp credentials are not given, the worker won't be started. This is not the case yet, currently amqp empty credentials are inserted even when you pick "none" on the GUI in Events tab.

In other words: I'd leave it like this for now.